### PR TITLE
Add test cases covering local symtab `imports`.

### PR DESCRIPTION
**Pull request by toddjonker**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: conformance/local_symtab_imports.ion [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+(ion_1_0 "Exact-match with larger max_id inserts padding"
+  (toplevel $ion_symbol_table::{imports:[{name:"empty", version:1, max_id:3},
+                                         {name:"abcs"}]}
+            '#$13')
+  (produces a))
+
+
+// Multiple imports
+
+// TODO:[TAB-BOUNDS] This should be easier.
```
Comment (popematt) [inline conformance/local_symtab_imports.ion:81]: What is `TAB-BOUNDS`?

**Review by popematt — APPROVED** [review]

**Review by tgregg — COMMENTED** [review]
Should we have the DSL declare that the catalog should be loaded, or are we saying the DSL implicitly loads the catalog?

**Comment by toddjonker** [comment]
@tgregg The (unwritten) intended behavior is that the catalog content (that is, all symtabs/modules in the `catalog` files) is implicitly available to all test cases in the `ion-tests` package. I don't see any reason why some cases would want to avoid that; there's no way to detect an empty catalog, and tests can use names that don't exist.  (We should probably specify one or two reserved names for that purpose.)

(Note that I didn't check whether any of these names are already in use outside the `conformance` directory.

File: conformance/local_symtab_imports.ion (hunk 1 above)
Comment (toddjonker) [reply conformance/local_symtab_imports.ion:81]: Oh, that's a label so I can find the test case when/if we add an expectation (to be proposed) for checking symbol table boundaries.  I've got a ton of those in the semantic model.

If you'd prefer, I can cut a placeholder issue and link to that instead; I just find this a nice lightweight mechanism.

**Review by tgregg — APPROVED** [review]

**Pull request merged**